### PR TITLE
Add ScreenTray.app v1.1.0

### PR DESCRIPTION
**Pull request by mwean**
https://screentray.com/

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/screentray.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'screentray' do
+  version '1.1.0'
+  sha256 'c014c85574196fd45550ebe964047d9e385bb405f7631f51814f944aa1a46210'
+
+  url 'https://screentray.com/download'
```
Comment (vitorgalvao) [inline Casks/screentray.rb]: `https://github.com/DSnopov/screentray-distribution/releases/download/v1.1.0/screen-tray-1.1.0-mac.zip`

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by mwean** [comment]
@vitorgalvao I updated the url

**Comment by reitermarkus** [comment]
Checksum has changed.

**Comment by mwean** [comment]
I just downloaded it from the site, and the checksum is the same.

**Comment by reitermarkus** [comment]
> Should I update the URL or the checksum?

Let's use the DMG URL then.

**Comment by mwean** [comment]
Done

**Pull request merged**